zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -55,7 +55,7 @@
 
 SERVICE_ACCOUNT_PATH = '/var/run/secrets/kubernetes.io/serviceaccount'
 
-SKIPPED_ANNOTATIONS = {'kubernetes.io/created-by'}
+SKIPPED_ANNOTATIONS = {'kubernetes.io/created-by', 'autoscaling.alpha.kubernetes.io/metrics', 'autoscaling.alpha.kubernetes.io/conditions'}
```

Comment:
is it `/metrics` only or `/current-metrics` as well?

Reply:
Fixed.